sdks/java/extensions/ml/src/test/java/org/apache/beam/sdk/extensions/ml/RecommendationAICatalogItemIT.java

```diff
@@ -42,13 +44,14 @@
 @RunWith(JUnit4.class)
 public class RecommendationAICatalogItemIT {
   @Rule public TestPipeline testPipeline = TestPipeline.create();
+  static String randomId = Integer.toString(new Random().nextInt());
```

Comment:
Nit: this doesn't need to be static, but should be private/final. (You should make catalogItem into a private final variable as well.)

```suggestion
  private final String randomId = Integer.toString(new Random().nextInt());
```